docs/changelog/84750.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 81272
+summary: spelling mistake in edge_ngram tokenizer name
+area: Asia/China
+type: enhancement
+issues: []
```

Comment:
We don't need a changelog entry for a change in an exception message, can you please remove this file?